### PR TITLE
Display config for the "Can't find the address?" button

### LINKED ISSUE #9: [Address lookup] "Can't find the address?" button is always visible
Rich has pointed out that the "Can't find the addres?" button is visible even before any address searches have been performed.  Screenshot attached:
screenshot-address-lookup-webform-element.

Perhaps it would be better if it appears after an **unsuccessful** search.  I am happy to implement the change.  But thought it better to consult first.  So, is this going to be acceptable with you @andrew-wallis @andybroomfield 

@richardaclarke @tom-steel

### PR DESCRIPTION
**Pull request by Adnan-cds**
Resolves #9

This change displays the manual address entry button labelled "Can't find the address?" based on address lookup element configuration.

The manual address entry button is displayed at all times at the moment.  This new configuration captures whether to reveal this button from the beginning or only after an address search.

By **default**, the manual address entry button will continue to appear as soon as the LocalGov address lookup element is loaded.

## How to test
- Install the localgov_forms module.
- Add the "LocalGov Address lookup" element to a Webform.
- The address lookup element's configuration form has a new config item called "When to display the manual address entry button".  The default is to **always** display it as is now the case.  Either leave it as it is or choose the other option.
- Save and test the form.

Screenshot of the new config item follows:
screenshot-address-lookup-config

### CODE REVIEW

**Comment by Adnan-cds** [comment]
Taking it to Draft state while we gather feedback from further manual testing.  Sorry about the confusion.

**Comment by Adnan-cds** [comment]
This is Ready for review.  We have agreed to add more tickets and pull requests to address the issues arising from the UX review.  So this one can go ahead as is.

**Review by ekes — COMMENTED** [review]
I'd say get it merged. Only reason to hold off is if @andybroomfield you think this would be an issue for your usage of this?

File: js/address_select.js [hunk 1]
```diff
@@ -29,28 +30,35 @@
...
       // Clear all values.
       manualAddressContainer.find('input').val('');
 
       // Trigger change events so UPRN and extra fields are cleared.
       manualAddressContainer.find('input').first().trigger('change');
     }
-    manualButton.removeClass('hidden');
+
+    // Hide manual entry button if asked for.
+    if (typeof settings.centralHub.isManualAddressEntryBtnAlwaysVisible !== 'undefined' && !settings.centralHub.isManualAddressEntryBtnAlwaysVisible) {
```
Comment (andybroomfield) [inline js/address_select.js:59]: Should probbably remove reference to CentralHub at some point. :-)

**Review by andybroomfield — APPROVED** [review]
Yes this is fine, we can pick up the issue about overriding the label and settings as a later issue.

**Comment by Adnan-cds** [comment]
Thanks Andy.  Merging...

**Pull request merged**